core/src/execution/jni_api.rs

```diff
@@ -216,18 +216,11 @@ fn prepare_datafusion_session_context(
         }
     }
 
-    // Get Datafusion configuration from Spark Execution context
-    // can be configured in Comet Spark JVM using Spark --conf parameters
-    // e.g: spark-shell --conf spark.datafusion.sql_parser.parse_float_as_decimal=true
```

Comment:
Could you keep this comment? It doesn't related to how the configs are read into session config but a general description what they come from.

Reply:
Okay

Reply:
added them in commit dd573e2354e47f1c6bb667a3942c617a08c1afc8